ccloader/js/mod.js

```diff
@@ -149,6 +179,50 @@ export class Mod {
 		this.runtimeAssets[original] = newPath;
 	}
 
+	/**
+	 * Returns the full path to a resource
+	 * @param {string} relativePath to resource with mod folder as base
+	 * @returns {string} 
+	 */
+	resolvePath(relativePath) {
+		const basePath = this._getBaseName(this.file); 
+		if (!relativePath) {
+			return this._normalizePath(basePath);
+		}
+		return this._normalizePath(basePath + '/' + relativePath);
+	}
+
+
+	/**
+	 * 
+	 * @param {string} relativePath to resource with mod folder as base
+	 * @returns {boolean} 
+	 */
+	hasResource(relativePath) {
+		if (this.loaded) {
```

Comment:
Should be changed to `if (!this.loaded) {return false}` to match the style of the other functions